pkg/support/dump.go

```diff
@@ -293,7 +293,11 @@ func (d *agentDumper) DumpAgentInfo(basedir string) error {
 }
 
 func (d *agentDumper) DumpMemberlist(basedir string) error {
-	return dumpAntctlGet(d.fs, d.executor, "memberlist", basedir)
+	err := dumpAntctlGet(d.fs, d.executor, "memberlist", basedir)
+	if err != nil && strings.Contains(err.Error(), "error when dumping memberlist:") {
```

Comment:
This would just ignore all errors as error returned by dumpAntctlGet always include this string. Can it get the target message "memberlist is not running" or the target status code `http.StatusServiceUnavailable` from the error in some way?

Reply:
dumpAntctlGet returns broadly two kinds of error -

1A. error due to memberlist not running , status code `http.StatusServiceUnavailable`.
1B. error when encoding Memberlist to json, status code `http.StatusInternalServerError`
These two cases mentioned above will fall in this if case.   

2. error when writing memberlist to file.

This case will NOT fall in this if case.

@tnqn Do you suggest to catch only case 1A in this if case (which writes no data to file)?

Reply:
Only 1A indicates things work correctly, so we should process 1A like success.